components/automate-chef-io/content/docs/SAML.md

```diff
@@ -0,0 +1,133 @@
++++
+title = "SAML"
+description = "SAML Configuration"
+date = 2018-05-11T09:27:09+00:00
+draft = false
+bref = ""
+toc = true
+[menu]
+  [menu.docs]
+    parent = "configuring_automate"
+    weight = 50
++++
+
+## Authentication via Existing Identity Management Systems
+
+Chef Automate can integrate with existing LDAP or SAML services authenticate users in Chef Automate, and thus use their existing group memberships to determine their Chef Automate permissions.
```

Comment:
I see that you've mostly duplicated this section between the new SAML.md and ldap.md, which is fine. I am making a few suggestions on both files, though, that take advantage of that separation and I think increase clarity.

```suggestion
Chef Automate can integrate with existing SAML services to authenticate users in Chef Automate, and thus use their existing group memberships to determine their Chef Automate permissions.
```